client/state/selectors/index.js

```diff
@@ -1,286 +1,18 @@
-export isRequestingJetpackUserConnection from './is-requesting-jetpack-user-connection';
-export isRequestingMedia from './is-requesting-media';
-export isRequestingMediaItem from './is-requesting-media-item';
-export isRequestingMissingSites from './is-requesting-missing-sites';
-export isRequestingPostLikes from './is-requesting-post-likes';
-export isRequestingReaderTeams from './is-requesting-reader-teams';
-export isRequestingResetPassword from './is-requesting-reset-password';
-export isRequestingSharingButtons from './is-requesting-sharing-buttons';
-export isRequestingSiteConnectionStatus from './is-requesting-site-connection-status';
-export isRequestingSiteMonitorSettings from './is-requesting-site-monitor-settings';
-export isRequestingWhois from './is-requesting-whois';
-export isRewindActivating from './is-rewind-activating';
-export isRewindActive from './is-rewind-active';
-export isRtl from './is-rtl';
-export isSavingSharingButtons from './is-saving-sharing-buttons';
-export isSchedulingPublicizeShareAction from './is-scheduling-publicize-share-action';
-export isSchedulingPublicizeShareActionError from './is-scheduling-publicize-share-action-error';
-export isSendingBillingReceiptEmail from './is-sending-billing-receipt-email';
-export isSharingButtonsSaveSuccessful from './is-sharing-buttons-save-successful';
-export isSiteAutomatedTransfer from './is-site-automated-transfer';
-export isSiteBlocked from './is-site-blocked';
-export isSiteOnFreePlan from './is-site-on-free-plan';
-export isSiteOnPaidPlan from './is-site-on-paid-plan';
-export isSitePressable from './is-site-pressable';
-export isSiteSupportingImageEditor from './is-site-supporting-image-editor';
-export isSiteUpgradeable from './is-site-upgradeable';
-export isTracking from './is-tracking';
-export isTransientMedia from './is-transient-media';
-export isTwoStepEnabled from './is-two-step-enabled';
-export isTwoStepSmsEnabled from './is-two-step-sms-enabled';
-export isUpdatingJetpackCredentials from './is-updating-jetpack-credentials';
-export isUpdatingJetpackSettings from './is-updating-jetpack-settings';
-export isUpdatingSiteMonitorSettings from './is-updating-site-monitor-settings';
-export isUserRegistrationDaysWithinRange from './is-user-registration-days-within-range';
-export isUserSettingUnsaved from './is-user-setting-unsaved';
-export isValidatingAccountRecoveryKey from './is-validating-account-recovery-key';
-export isValidThemeFilterTerm from './is-valid-theme-filter-term';
-export isVipSite from './is-vip-site';
-export isWordpressUpdateSuccessful from './is-wordpress-update-successful';
-export prependThemeFilterKeys from './prepend-theme-filter-keys';
-export shouldShowVideoEditorError from './should-show-video-editor-error';
-export shouldSyncReaderFollows from './should-sync-reader-follows';
-export userHasAnyPaidPlans from './user-has-any-paid-plans';
+throw new Error(
```

Comment:
Patch updated to make import w/ a path break. I still put a `throw` here though, because our transpiler will prevent the `import` line from throwing — a non-existent selector function only throw when it's called. A `throw` here makes it throw when Calypso loads.